### PR TITLE
Replace old workstation docs with new workstation docs

### PR DESCRIPTION
**Pull request by IanMadd**
Signed-off-by: IanMadd <imaddaus@chef.io>

This puts the workstation documentation into chef-workstation so we can pull it back into chef-web-docs whenever chef-workstation is promoted.

## Description

This PR:

- adds workstation documentation from the chef-web-docs repo
- replaces the original hugo theme with the new hugo theme
- customizes the index page to make it easier to write docs from the chef-workstation repo
- `about_workstation.md` has been renamed `_index.md` and there's a new `workstation/single.html` template. Users who go to `localhost:1313/workstation/` or `docs.chef.io/workstation/` will see the About Workstation page.
- all shortcodes in `layouts/shortcodes/` have a `ws_` prefix so we can distinguish them from shortcodes in chef-web-docs from other sources.

To Do:

- There were several pages in the old chef-workstation docs that aren't in chef-web-docs. I don't know where to put them so they're unedited in `www/content/workstation/`. Some of it is chef-run docs, some is content that could be deleted or edited into pages from chef-web-docs.

## Related Issue

chef/release-engineering#983

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [X] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: www/README.md [hunk 1]
```diff
@@ -66,65 +48,53 @@ in [Markdown](https://en.wikipedia.org/wiki/Markdown).
...
 |   ├── static
 |   |   ├── images
 |   |   |   ├── chef-workstation    # where to keep any images you need to reference in your documentation
+|   |   ├── css
 ```
 
 ### What is happening behind the scenes
 
-Everytime a PR is merged into `chef/chef-workstation`, the contents of the `chef-workstation` directories from the `site` directory are synced to `chef/chef-www` and the `chef-www-acceptance.cd.chef.co` acceptance environment is automatically updated. This is handled by the Expeditor subscriptions in the `chef/chef-www` GitHub repository.
+TODO
```
Comment (tyler-ball) [inline www/README.md]: @IanMadd I'll need some help with the TODOs here - I'm not sure of the process of how our docs get deployed

File: www/content/workstation/getting_started.md [hunk 2]
```diff
@@ -2,14 +2,14 @@
 title = "Getting Started"
 draft = false
 
-aliases = ["getting_started.html", "/workstation_setup.html", "/chefdk_setup.html", "/workstation.html", "/getting_started/"]
```
Comment (tyler-ball) [inline www/content/workstation/getting_started.md]: In `install_workstation.md` I linked to `[Getting Started](/workstation/getting_started/)` - will that break now?

Comment (tyler-ball) [reply www/content/workstation/getting_started.md]: Nope! That link worked fine in my local build. Although http://localhost:1313/workstation/workstation_setup/ gives me a 404. I apparently do not understand what these aliases mean :)

Comment (IanMadd) [reply www/content/workstation/getting_started.md]: The alias adds a blank page at that url with a redirect to the page that specifies the alias.

**Pull request merged**